diff --git a/chef-bin/chef-bin.gemspec b/chef-bin/chef-bin.gemspec
index 66d14f3f5..832ca205f 100644
--- a/chef-bin/chef-bin.gemspec
+++ b/chef-bin/chef-bin.gemspec
@@ -22,5 +22,5 @@ Gem::Specification.new do |spec|
     Dir.glob("{lib}/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) }
 
   spec.bindir = "bin"
-  spec.executables = %w{ chef-apply chef-client chef-resource-inspector chef-service-manager chef-shell chef-solo chef-windows-service }
+  spec.executables = %w{ cinc-apply cinc-client cinc-resource-inspector cinc-service-manager cinc-shell cinc-solo cinc-windows-service }
 end
