--- b/lib/chef/dist.rb	2019-04-19 21:00:51.529341878 +0000
+++ a/lib/chef/dist.rb	2019-04-19 21:02:11.652087582 +0000
@@ -1,18 +1,18 @@
 class Chef
   class Dist
     # This class is not fully implemented, depending on it is not recommended!
     # When referencing a product directly, like Chef (Now Chef Infra)
-    PRODUCT = "Chef Infra Client".freeze
+    PRODUCT = "Cinc Client".freeze
 
     # The client's alias (chef-client)
-    CLIENT = "chef-client".freeze
+    CLIENT = "cinc-client".freeze
 
     # The chef executable, as in `chef gem install` or `chef generate cookbook`
-    EXEC = "chef".freeze
+    EXEC = "cinc".freeze
 
     # product website address
-    WEBSITE = "https://chef.io".freeze
+    WEBSITE = "https://gitlab.com/cc-build/organization-subjects/wikis/home".freeze
   end
 end
