diff --git a/lib/chef/dist.rb b/lib/chef/dist.rb
index ed9307900..119a4551e 100644
--- a/lib/chef/dist.rb
+++ b/lib/chef/dist.rb
@@ -1,19 +1,19 @@
 class Chef
   class Dist
     # This class is not fully implemented, depending on it is not recommended!
-    # When referencing a product directly, like Chef (Now Chef Infra)
-    PRODUCT = "Chef Infra Client".freeze
+    # When referencing a product directly, like Chef (Now Cinc)
+    PRODUCT = "Cinc Client".freeze
 
     # The name of the server product
-    SERVER_PRODUCT = "Chef Infra Server".freeze
+    SERVER_PRODUCT = "Cinc Server".freeze
 
     # The client's alias (chef-client)
-    CLIENT = "chef-client".freeze
+    CLIENT = "cinc-client".freeze
 
     # The chef executable, as in `chef gem install` or `chef generate cookbook`
-    EXEC = "chef".freeze
+    EXEC = "cinc".freeze
 
     # product website address
-    WEBSITE = "https://chef.io".freeze
+    WEBSITE = "https://gitlab.com/cc-build/organization-subjects/wikis/home".freeze
   end
 end
