From b00c4f5e22cb4c0c6e185a203f6a57616c9d8d80 Mon Sep 17 00:00:00 2001
From: Lance Albertson <lance@osuosl.org>
Date: Sat, 5 Oct 2019 13:10:38 -0700
Subject: [PATCH] Stable Cinc patches

---
 config/software/chef.rb | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/config/software/chef.rb b/config/software/chef.rb
index 42f8375..3bcb17f 100644
--- a/config/software/chef.rb
+++ b/config/software/chef.rb
@@ -1,5 +1,6 @@
 #
 # Copyright 2012-2019, Chef Software Inc.
+# Copyright 2019, Cinc Project
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
@@ -110,4 +111,9 @@ build do
       remove_directory "#{gem_install_dir}/test"
     end
   end
+
+  copy "#{project_dir}/cinc/cinc-wrapper", "#{install_dir}/bin/"
+  %w(chef-apply chef-client chef-shell chef-solo).each do |bin|
+    link "#{install_dir}/bin/cinc-wrapper", "#{install_dir}/bin/#{bin}"
+  end
 end
