--- chef.gemspec        2019-04-05 22:29:20.853987293 +0000
+++ cinc.gemspec        2019-04-05 22:29:58.054498610 +0000
@@ -48,7 +48,7 @@
   s.add_dependency "bundler", ">= 1.10"

   s.bindir       = "bin"
-  s.executables  = %w{ chef-client chef-solo knife chef-shell chef-apply chef-resource-inspector }
+  s.executables  = %w{ cinc-client cinc-solo knife cinc-shell cinc-apply cinc-resource-inspector }

   s.require_paths = %w{ lib }
   s.files = %w{Gemfile Rakefile LICENSE README.md} + Dir.glob("{lib,tasks,spec}/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) } + Dir.glob("*.gemspec")
